### PR TITLE
Dao fix majorityvote issue

### PR DESCRIPTION
**Pull request by ManfredKarrer**
Fixes https://github.com/bisq-network/bisq/issues/2362

### CODE REVIEW

**Comment by ManfredKarrer** [comment]
Needs more testing before merge.

**Comment by ManfredKarrer** [comment]
@sqrrm I have tested now with different scenarios with blind vote missing, ignore merit for majorit hash case,... Could you give it a critical review and test cycle as well?

**Comment by sqrrm** [comment]
I am looking at this but it will take a while as it's rather complicated. @ripcurlx hope to be done before the next release as I think this is quite important to get in.

File: desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java [hunk 1]
```diff
@@ -214,6 +218,37 @@ private void onResultsListItemSelected(CycleListItem item) {
...
+                                    vote.isAccepted() ?
+                                            Res.get("dao.proposal.display.myVote.accepted") :
+                                            Res.get("dao.proposal.display.myVote.rejected");
+                            sb.append(Res.getWithCol("dao.results.votes.table.header.vote")).append(" ")
+                                    .append(voteString).append("\n");
+
+                        });
+                    });
+            if (!sb.toString().isEmpty()) {
+                new Popup<>().information("We had invalid votes in that voting cycle. That can happen if a vote was " +
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java]: String should be put into translations.

**Review by ripcurlx — COMMENTED** [review]
utACK

File: desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java (hunk 1 above)
Comment (ManfredKarrer) [reply desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java]: Upps, yes.

**Review by sqrrm — APPROVED** [review]
utACK

I added some refactoring as a PR to manfred's repo. Had to refactor to understand what's going on. I haven't tested this yet.

File: desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java (hunk 1 above)
Comment (ManfredKarrer) [reply desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java]: Fixed

**Comment by ManfredKarrer** [comment]
@sqrrm I will merge to not delay release, but please go on testing, its a very critical PR.

**Pull request merged**